mflike/mflike.py

```diff
@@ -246,25 +246,17 @@ def get_sacc_names(pol, exp_1, exp_2, freq_1, freq_2):
                 # The only reason why we need indices is the symmetrization.
                 # Otherwise all of this could have been done in the previous
                 # loop over data['spectra'].
-                ind = s.indices(dtype,
-                                (tname_1, tname_2))
+                ls, cls, ind = s.get_ell_cl(dtype, tname_1, tname_2, return_ind=True)
+                # ws = s.get_bandpower_windows(ind)
                 if cbbl_extra:
                     ind_b = s_b.indices(dtype,
                                         (tname_1, tname_2))
-
-                if cbbl_extra:
-                    ls, cls = s.get_ell_cl(dtype, tname_1, tname_2,
-                                           return_windows=False)
-                    _, _, ws = s_b.get_ell_cl(dtype, tname_1, tname_2,
-                                              return_windows=True)
-                else:
-                    ls, cls, ws = s.get_ell_cl(dtype, tname_1, tname_2,
-                                               return_windows=True)
+                    ws = s_b.get_bandpower_windows(ind_b)
```

Comment:
Humm, I think the intention here was that, if `cbbl_extra` then, the windows are read from that file (`s_b`), otherwise the assumption is that they reside in the default file (`s`). So I'd add an
```
else:
    ws = s.get_bandpower_windows(ind)
```
and then remove line 250.

Reply:
I think the window is only stored in the cov file https://github.com/simonsobs/PSpipe/blob/master/project/maps2params/python/maps_to_params_port_to_sacc.py#L198

Reply:
yep, so that will still happen even if you add this line. But this way you'll be able to run the likelihood (with a faster setup, since you don't have to read 2 files) if you want to provide everything in the same file. Up to you though!